src/style.css

```
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(images/star.svg);
+}
+
+.stars--5 .stars__star {
```

```suggestion
.stars--5 .stars__star,
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4) {
```
Consider using commas to group selectors